### DISCUSSION TITLE
Where can I find the latest ARM32 Windows Runtime version?

### DISCUSSION THREAD

**Discussion by kristibektashi** [ikvmnet/ikvm#476] [Q&A] [answered]
So I found these issues (https://github.com/ikvmnet/ikvm/issues/198 /https://github.com/ikvmnet/ikvm/pull/201) that mention runtime support for ARM32 Windows has been added to IKVM, however the latest release version doesn't seem to have an ARM32 Windows version available. The last version I could find for ARM32 Windows is 8.6.4. And since I didn't see anywhere mentioned that Windows ARM32 support was removed (and both .NET 4.7.2 and .NET 6.0 should still support ARM32), I decided to open this discussion thread so that I can ask: 

Has ARM32 Windows support been removed? 
If yes, I assume 8.6.4 is the last version to support ARM32 Windows, is that correct?
If not, where can I find the latest version available for ARM32 Windows? Or do I have to compile it myself?

**Comment by wasabii** [comment]
Runtime support should be working fine. Unless it's not.

It's not like I test that.

**Comment by wasabii** [comment]
Hmm. I'm not seeing the releases though. Let me look around and see what happened.

**Comment by wasabii** [comment]
Actually, I see that it was removed in 8.7, because ARM32 Windows is no longer supported on .NET Core 6. And apparently MS has no supported operating systems running ARM32 anymore.

Were we wrong?

**Reply by kristibektashi** [reply]
You aren't wrong in that there are no supported ARM32 Windows versions anymore, unless you count the ARM64 Windows systems that can also run ARM32 applications. And if you're not planning on re-adding ARM32 Windows support you might wanna mention somewhere that it was removed and redirect users who are looking for it (like me) to the last version that exists for it (aka 8.6.4).

However, both .NET 6 and 7 should still support ARM32 (support was removed in .NET 8), they just don't provide it in releases anymore. You have to install it yourself, I believe you can do so with Nuget (not entirely sure though). And .NET Framework should still work fine (I believe Windows RT supports .NET 4.8, still not entirely sure though)

**Reply by wasabii** [reply, marked as answer]
Yes. It should have been mentioned. I will go add a mention.

I do not think it's going to be added back.

Generally, unless there's a super compelling reason, I'm going to only track "supported things". Old versiosn exist, if people have a dire need, as you said. I dropped support for < NET472 for that reason: the last supported OS is Server 2012 R2 SP? which ships with NET472.

And though you can technically compile .NET 6 for ARM32, I think... it's not "supported" by MS.
